### ISSUE TITLE
[ENHANCEMENT] Improve documentation on updates

### ISSUE THREAD

**Issue by tobiastom** [BastilleBSD/bastille#434]
**Is your feature request related to a problem? Please describe.**
While the commands `update` and `upgrade` are documented itself, the process on how to upgrade releases and jails does not seem to be documented. 

**Describe the solution you'd like**
I'd like to see a separate documentation page, which describes the recommended way to upgrade release and their jails. While I'd like to contribute this page, I'd like to make sure the process was correct:

I did update a 12.2-RELEASE to a 13.0-RELEASE today, and did the following:

```bash
bastille upgrade 12.2-RELEASE 13.0-RELEASE # update the release
bastille upgrade 12.2-RELEASE install # install it
bastille upgrade 12.2-RELEASE install # twice, as the system should be rebooted, is this necessary? 
bastille pkg ALL upgrade -y # update the packages
bastille restart ALL # restart all jails
bastille upgrade 12.2-RELEASE install # install after reboot
```

Is there something missing or duplicate? 

`bastille cmd alertmanager freebsd-version` now tells me `13.0-RELEASE-p3`.

Unfortunately this is confusing:

```
root@kahlo:/usr/local/bastille/jails/alertmanager # cat fstab 
/usr/local/bastille/releases/12.2-RELEASE /usr/local/bastille/jails/alertmanager/root/.bastille nullfs ro 0 0
```

Is it correct that the directory `12.2-RELEASE` now contains the `13.0-RELEASE` release? 

How could I have updated the jails in a simpler way? Would it be enough to change the fstab entry to a bootstrapped `13.0-RELEASE` release? What other steps should I have done? `png upgrade` at least for the ABI changes.

Thanks for your help.

**Comment by lifepillar** [comment]
I second this request. Release upgrade is an important process: it should be documented well!

A comment about the second command:

```
bastille upgrade <OLD>-RELEASE install
```

At a first glance, when I saw this I thought: “shouldn't it be `<NEW>-RELEASE install`? But then I realized it's *upgrading* the old release (by installing the new one). If this issue is solved, I'd like also the documentation to explicit confirm that the release to be passed to `bastille upgrade … install` is the old one.

I followed these steps to upgrade from 13.0 to 13.1 and all went well. I still have only a `releases/13.0-RELEASE` directory. Could the name of the directory be updated as well?

**Comment by foudfou** [comment]
https://gist.github.com/cedwards/1e00c3d0aa6fbb14bc5b16fca8df0c35 ?

**Comment by lifepillar** [comment]
Thanks! I have recently upgraded from 13.1 to 13.2 using the suggested method (bootstrap 13.2-RELEASE then update the `fstab` files), and it workedly flawlessly. The procedure makes sense to me, and it also lets me decide which jails to upgrade. So, I have personally deprecated `upgrade` :-)

I also hope that the gist will be moved to the official documentation!

**Comment by bmac2** [comment]
this was merged to master with PR #590.  Closing as completed.
